example_app_generator/generate_app.rb

```diff
@@ -33,6 +33,10 @@
     append_to_file('Gemfile', "gem 'rails-controller-testing'\n")
   end
 
+  if Rails::VERSION::STRING ~= /^4\./
```

Comment:
We don't need to test against 4 anymore, so this should be at least tagged with a todo to remove

Reply:
Do you think we should "allow failures" or remove from the build matrix Rails 4.2?

Reply:
When @pirj's PR #2226 is re-opened it should remove them from the build matrix.

Reply:
Perfect. Thanks

Reply:
I'll handle it this week.

Reply:
It should not be needed anymore. https://github.com/rails/rails/issues/37978

Reply:
Removed in the last push